test/BUILD.gn

```diff
@@ -361,8 +361,19 @@ test("brave_unit_tests") {
   }
 
   if (enable_ai_chat) {
-    sources += [ "//brave/components/ai_chat/core/ai_chat_metrics_unittest.cc" ]
-    deps += [ "//brave/components/ai_chat/core" ]
+    sources += [
+      "//brave/components/ai_chat/core/ai_chat_credential_manager_unittest.cc",
```

Comment:
please do not add individual test sources to `test/BUILD.gn`. You should add a test target in the component and add a dep here for it (and remove the other deps)